src/main/proto/netflix/titus/titus_job_api.proto

```diff
@@ -781,6 +803,11 @@ service JobManagementService {
     rpc UpdateJobCapacity (JobCapacityUpdate) returns (google.protobuf.Empty) {
     }
 
+    /// Modify job capacity for a service job. It allows you to specify only values (min / max / desired) that need to be updated.
```

Comment:
Should the comment give some guidance for which one (UpdateJobCapacity vs UpdateJobCapacityValues) a user should use? I could see it being confusing.